packages/auth-nextjs/src/app/index.ts

```diff
@@ -135,14 +135,33 @@ export class NextAppAuth extends NextAuth {
         return tokenData;
       },
       emailPasswordResendVerificationEmail: async (
-        data: FormData | { verification_token: string }
+        data: FormData | { verification_token: string } | { email: string }
```

Comment:
I kept this in single action, as adding another action would require more docs changes, and potentially would be confusing to have 2 calls for 1 endpoint